LICENSE-3rdparty.csv

```diff
@@ -63,6 +64,7 @@ dev,globals,MIT,Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (https://si
 dev,graphql,MIT,Copyright 2015 Facebook Inc.
 dev,jszip,MIT,Copyright 2015-2016 Stuart Knightley and contributors
 dev,knex,MIT,Copyright (c) 2013-present Tim Griesser
+dev,koa,MIT,Copyright (c) 2019 Koa Contributors
```

Comment:
Any way to test `koa` support without having to explicitly depend on it in our devDependencies?

Reply:
It would require more substantial changes.

Reply:
Or we could use a own mocked module that we instrument. I think it just makes sense to rely on the real modules in that case.

Reply:
Definitely makes sense to rely on the real module. However, if we were to do that, we'd end up with dependencies on every single module we instrument. That also makes it impossible to test multiple versions easily. That's why I wonder if something can be done quickly to avoid this, and if not, we should definitely come up with a plan to address this as soon as possible.